PatientWebApplication/UserMicroserviceApi/Model/Entity.cs

```diff
@@ -0,0 +1,18 @@
+/***********************************************************************
+ * Module:  Entity.cs
+ * Author:  Tamara
+ * Purpose: Definition of the Class Repository.Entity
+ ***********************************************************************/
+
+namespace UserMicroserviceApi.Model
+{
+    public class Entity
+    {
+        public Entity() { }
+        public Entity(int id)
+        {
+            this.id = id;
+        }
+        public int id { get; set; }
```

Comment:
Change property name to start with upper letter and apply in rest of code (remove 'this' from constructors).
Set property above constructor.